client/src/research/ResearchPage.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import Interactions from './Interactions';
+
+
+// The page for logged-in user to access research data.
```

Comment:
@gballard1 This is a placeholder page that will ultimately have to check that the user is authorized, and then will show the visualizations you've been working on in `Interactions`.